include/libcyphal/transport/session/msg_sessions.hpp

```diff
@@ -29,13 +29,16 @@ struct MessageTxParams final
 class IMessageRxSession : public IRxSession
 {
 public:
-    CETL_NODISCARD virtual MessageRxParams getParams() const noexcept = 0;
+    CETL_NODISCARD virtual MessageRxParams   getParams() const noexcept = 0;
+    CETL_NODISCARD virtual MessageRxTransfer receive()                  = 0;
```

Comment:
There's a missing detail in the class diagram although it is deducible from the following description: all `receive` methods should return an optional. Here it would be `cetl::optional<MessageRxTransfer>`. All methods are non-blocking; if there's nothing to return, an empty option is used.